cli/prisma2/src/bin.ts

```diff
@@ -91,7 +93,20 @@ async function main(): Promise<number> {
     console.error(result)
     return 1
   }
-  console.log(result)
+  // check prisma for updates
+  const checkResult = await checkpoint.check({
+    product: 'prisma',
+    version: packageJson.version,
+    disable: isCI,
+  })
+  // if the result is cached and we're outdated, show this prompte
+  if (checkResult.status === 'ok' && checkResult.data.outdated) {
+    console.error(
+      `\n${chalk.blue('Update available')} ${packageJson.version} -> ${
+        checkResult.data.current_version
+      }\nRun ${chalk.bold(checkResult.data.current_download_url)} to update`,
```

Comment:
Random dinner thought:
This might be a lot more complex. `npm` vs. `yarn`, local vs. global, `npx`...

Reply:
we can also pass that information in the request to tailor the response.

we have the option where we can say either the client knows how to install itself or the server tells it how. I went with the server because all the semver logic is there, but could just as easily see the client knowing how.